app/views/layouts/_header.html.erb

```diff
@@ -0,0 +1,12 @@
+<div>
+  <div class="header-container">
+    <span class="header-title"><%= link_to('The Library', root_path) %></span>
+    <div class="header-links">
+      <%= link_to('My Requests', requests_path) %>
+      <% if(current_user) %>
+        <%= link_to('Log Out', destroy_user_session_path, :method => :delete) %>
```

Comment:
Use the new Ruby hash syntax. `method: :delete`

Also you can avoid () if you like.